03/app.js

```diff
@@ -1,7 +1,12 @@
 export default function randomNumber(min, max) {
-    if(typeof min !== 'number' || typeof max !== 'number') {
-        throw new TypeError('Given parameter is not a number')
+    if(typeof min !== 'number') {
```

Comment:
Jeszcze uważałbym na `NaN` - to też jest `number`, ale raczej nie o takie `number` nam chodzi ;P